src/style.css

```diff
@@ -1,3 +1,75 @@
-body {
+html {
+  font-family: Roboto, sans-serif;
+
+  --text-color:#000;
+  --hover-color:#00ACDC;
+  ;
+}
+
+body, div, p, h1, h2, h3,a,li,ul,nav {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  width: 100vw;
```

Comment:
tag `header` is block element and by default stretches across the entire width
so this rule is unnecessary